tests/utils/compare.js

```diff
@@ -16,7 +16,7 @@ function sendReference (filename, data) {
   const req = new XMLHttpRequest()
   req.open('POST', `http://localhost:9090/${filename}`, true)
   req.onload = e => {
-    console.log(e)
+    //console.log(e)
```

Comment:
@MrRio 
I commented this out to run again the failed test (which failed because of a disconnected test). I guess this is a valid change too. It was adding unecessary output to the travis-log ("Object {}").